docs/getting-started/code-sandbox.md

```diff
@@ -0,0 +1,3 @@
+# Code sandbox
+
+If you want to play around with unit testing without installing or bootstrapping a project, [here's a code sandbox template](https://codesandbox.io/s/roku-unit-testing-16c26?file=/tests/source/utils.test.brs) you can use.
```

Comment:
What's involved in updating the sandbox?  I imagine we'll need to change that one day.  Is it a codepen kind of thing where the URL includes a hash of the source, and if the sources change we just update the URL here?

Reply:
Hmmm not quite a hash, I don't think. The base link wouldn't change if the sandbox gets updated, but the url parameter with the file name might need to change if the file got removed. I only added that parameter because I thought it was a nicer entry point than the default `package.json`, but we can always change it 🤷‍♂️